### ISSUE TITLE
Update docs, please!

### ISSUE THREAD

**Issue by whitesmoke5** [jenv/jenv#337]
Yet another open source repo fallen victim to a poorly maintained docs site. Word of advice if you want people to adopt your wares, then I suggest keeping the docs as up-to-date as your code base. Please, make your docs align with current updates to related tech.

**Comment by gausam** [comment]
Good point. Can you file your solution/recommendation as a pull request? That would definitely save the community some hours.
